requirements.txt

```diff
@@ -3,3 +3,5 @@ iso8601
 phonenumbers
 python-magic
 requests
+py3o.template
```

Comment:
Il ne faut pas les ajouter ici, mais dans le fichier "__manifest__.py".
Le fichier requirements est autogenere a partir de tous les manifestes.

Reply:
Generation faite par ton pre-commit by the way.

Reply:
Nota : si par hasard, le import n'a pas la même valeur que le pip install tu peux mettre la valeur de l'import dans le fichier ``__manifest__.py`` et la valeur du pip install dans le fichier ``setup/MODULE_NAME/setup.py``

Exemple pour magic / python-magic : 
- import = ``magic`` : https://github.com/commown/commown-odoo-addons/blob/12.0/commown/__manifest__.py#L55
- pip install = ``python-magic`` : https://github.com/commown/commown-odoo-addons/blob/12.0/setup/commown/setup.py#L6

Reply:
Merci !